service/history/consts/const.go

```diff
@@ -69,13 +69,21 @@ var (
 	// ErrSignalsLimitExceeded is the error indicating limit reached for maximum number of signal events
 	ErrSignalsLimitExceeded = serviceerror.NewInvalidArgument("exceeded workflow execution limit for signal events")
 	// ErrWorkflowClosing is the error indicating requests to workflow can not be applied as workflow is closing
-	ErrWorkflowClosing = serviceerror.NewResourceExhausted(enumspb.RESOURCE_EXHAUSTED_CAUSE_BUSY_WORKFLOW, "workflow operation can not be applied because workflow is closing")
+	ErrWorkflowClosing = serviceerror.NewResourceExhausted(
+		enumspb.RESOURCE_EXHAUSTED_CAUSE_BUSY_WORKFLOW,
+		enumspb.RESOURCE_SCOPE_SYSTEM,
```

Comment:
This is a namespace-level issue, not a system-level one (cc @yycptt)

Reply:
Yeah. Made the change.